rustic-cargo.el

```diff
@@ -65,6 +65,19 @@ If nil then the project is simply created."
   :type 'boolean
   :group 'rustic-cargo)
 
+(defcustom rustic-cargo-use-last-stored-arguments nil
+  "Always rerun cargo commands with stored arguments.
+
+Example:
+When turned off, running `rustic-cargo-test' with `C-u' will store the
+arguments in `rustic-test-arguments' and allows you to reuse them with
+`rustic-cargo-test-rerun'.
```

Comment:
This description is a bit confusing.

When `rustic-cargo-use-last-stored-arguments` is `nil`, then `rustic-cargo-test` will always use `rustic-default-test-arguments`.

Should the part about `C-u M-x rustic-cargo-test` be described for the function `rustic-cargo-test` but not for this `rustic-cargo-use-last-stored-arguments` variable?

Reply:
> Should the part about C-u M-x rustic-cargo-test be described for the function rustic-cargo-test but not for this rustic-cargo-use-last-stored-arguments variable?

We should apply this variable also for other cargo commands that have rerun commands. I think it's too much if we add a sentence to every command.

Reply:
Thanks for the review, I applied your suggestion.

Reply:
@brotzeit: thanks for your effort! :+1: